Firestore/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
   is IN a specified array. `whereField(_:arrayContainsAny:)` finds documents
   where a specified field is an array and contains ANY element of a specified
   array.
+- [changed] Firestore SDK now uses Nanopb rather than Objective-C Protobuf
+  library for parsing protos. This change does not affect visible behavior of
+  the SDK in any way. While we don't anticipate any issues, please [report any
+  issues with network behavior or
+  persistence](https://github.com/firebase/firebase-ios-sdk/issues/new) that you
+  experience. (#4013, #4116)
```

Comment:
I’m on the fence about whether or not these links are useful. The issue is that if someone comes to the release notes looking for clues about a problem they’re having these PRs aren’t really going to help. If you disagree, feel free to keep them.

Reply:
Removed.